src/front/js/pages/home.js

```diff
@@ -1,26 +1,17 @@
 import React, { useContext } from "react";
-import { Context } from "../store/appContext";
-import rigoImageUrl from "../../img/rigo-baby.jpg";
+// import { Context } from "../store/appContext";
```

Comment:
Unused import? Let's clean this comment here, then! 🧹

Reply:
+1